internal/database/sqlcommon/operation_sql.go

```diff
@@ -180,15 +180,21 @@ func (s *SQLCommon) UpdateOperation(ctx context.Context, id *fftypes.UUID, updat
 		return err
 	}
 
+	for _, hook := range hooks {
+		s.postCommitEvent(tx, hook)
+	}
+
 	return s.commitTx(ctx, tx, autoCommit)
 }
 
-func (s *SQLCommon) ResolveOperation(ctx context.Context, id *fftypes.UUID, status fftypes.OpStatus, errorMsg string, output fftypes.JSONObject) (err error) {
+func (s *SQLCommon) ResolveOperation(ctx context.Context, ns string, id *fftypes.UUID, status fftypes.OpStatus, errorMsg string, output fftypes.JSONObject) (err error) {
```

Comment:
Feels slightly wrong that `ns` makes its way to the change event but _not_ to the update filter. Should we ensure that the operation being updated _is_ actually in this namespace?

Reply:
Going to address this in a new PR, from a fork, as I can't rebase this branch.